language-design.md

```diff
@@ -17,7 +12,49 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> "After all, the very existence
+of a master plan means, by definition, that the members of the community
+can have little impact on the future shape of their community, because most
+of the important decisions have already been made. In a sense, under a master
+plan people are living with a frozen future, able to affect only relatively 
+trivial details." [Steele, 1998]
+
+This quote by Christopher Alexander is something I definitely agree with. Before
+reading this article, I did not think about this much and I would have maybe 
+held the opinion that it would be good to have a master plan and have the team
+working on the language determine all aspects of the design. However, I thought
+about the Go community in particular and how they are very transparent and
+take suggestions and implement them in the language regularly. This creates,
+of course, a great community, but also a decent indicator as to what features
+are needed generally versus more "niche features" as mentioned in the Steele
+article.
+
```

Comment:
Great points.  Before the readings, I too imagined that the process of designing a language would be less reliant on users.  But I wonder where you should draw the line.  Do you think it is feasible to have a language where users themselves could add features (that are, perhaps, first vetted by a committee)?